script/main.js

```diff
@@ -149,16 +149,15 @@ async function createNote(){
     let titleField = document.getElementById("title-field").value;
     let textAreaField = document.getElementById("text-area-field").value;
     
-    const response = await fetch(`${APIUrl}/submit` , {
+    try{
+        const response = await fetch(`${APIUrl}/submit` , {
         method:'POST',
         headers: {
             'Content-Type': 'application/json',
         },
         body: JSON.stringify({title: titleField, description: textAreaField})
-    });
-
-    if(!response.ok){
-        throw new Error('There is a problem.');
+    })}catch(error){
+            alert("There was an error. Please try again.", error)
```

Comment:
de error zelf tonen we liever niet aan de user. want dit komt van de backend en kan heel technisch zijn. de user is daar niets mee. 
je kan de alert houden hoe die is. zodat de user weet dat er iets mis is gegaan.
en de error zelf kan je in een ```console.error(error)``` zetten.